### ISSUE TITLE
[Chrome] Opening Chrome moves the last tab on the right next to the tab you were on when closing Chrome

### ISSUE THREAD

**Issue by PuppetMaster000** [autonome/alwaysright#35]
This happens consistently, every time you close and reopen Chrome, the last tab will be moved right of the tab you were on when you close Chrome. Closing and reopening the browser will move another one, and so on.
If it makes it easier with numbers, lets say you have 5 tabs open and you are on tab 2, tabs would look like this originally:
1,2,3,4,5
When you close Chrome and reopen it, suddenly it looks like this:
1,2,5,3,4
Then if you close and reopen again, it goes:
1,2,4,5,3 and so on.

**Comment by autonome** [comment]
does the active tab (2) remain the same each restart?

**Comment by autonome** [comment]
maybe regression from the fix for #34

**Comment by PuppetMaster000** [comment]
> does the active tab (2) remain the same each restart?

Yes, as far as I could tell, the only part that moves is the very last tab. I may be wrong on this but that is how it seems.
I also believe it has something to do with the changes from the previous fix

**Comment by autonome** [comment]
Startup on Firefox used to be a non-deterministic set of events, and this stuff would happen - like sometimes the ENTIRE tab strip would be REVERSED after startup because of this addon.

But then they changed startup to be a deterministic set of events, and these problems went away.

IIRC I had to do stuff like wait a bit after startup, even just a second or two. 

I'll first see if I can reproduce.

**Comment by PuppetMaster000** [comment]
Any progress on this?

**Comment by autonome** [comment]
i'll try and look this coming week

**Comment by gioxx** [comment]
I've been having the same issue, and it happens with both single tabs (but not often) and tab groups.
Usually, each time I reopen Chrome, it takes a closed tab group (which I always keep at the end of the open tab row) and moves it to the beginning.

**Comment by autonome** [comment]
I wasn't able to reproduce this problem.

If you can find some way to trigger this, it would help me know what to fix

**Comment by gioxx** [comment]
I can totally record a screencast and try to recreate the same scenario I have in Chrome, in the profile I use every day. Otherwise, I'm afraid I'm a bit useless to help you 😄

**Comment by autonome** [comment]
submitted a new version to chrome store with some fixes.

deterministic initialization at startup, not doing any moves until first active tab event.

will bump once it's approved so you can test.

**Comment by annitas5716-del** [comment]
The issue no longer appears in my environment. It's resolved. Thank you. :)

**Comment by autonome** [comment]
Thanks for confirming!

Fixed by [dcd9f68](https://github.com/autonome/alwaysright/commit/dcd9f687726b682a37b0e00774db7a4ab586bb54)
